charley_tran/week_02/catexercise.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <meta charset="utf-8" />
+  <title>Cat Walk</title>
+ </head>
+ <body>
+  
+  <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif">
+  
+
+
+ </body>
+ <script>
+var catGif = document.getElementsByTagName('img')[0];
+
+catGif.style.left = '0px';
+var x = 0 //defining starting position
+
+function catWalk() { //for cat walking fowards
+	catGif.style.left = x + "px";
+	x += 10;
+	if (catGif.style.left == "500px") { //once it hits halfway, stops the movement, chaneges picture.
```

Comment:
Think how you can make this center flexible so cat can walk to the middle of any screen